doc/manual/local.mk

```
@@ -1,5 +1,9 @@
 ifeq ($(doc_generate),yes)
 
+# Technically it will be defined by mk/programs.mk
+# but we can't use it as dependency of target unless we enable .SECONDEXPANSION:
+nix_PATH = $(buildprefix)src/nix/nix
```

It can be made more generic with
```suggestion
nix_PATH = $(buildprefix)$(nix_DIR)/nix
```